src/components/AccountDetails/Copy.tsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react'
-import styled from 'styled-components'
+import styled from 'styled-components/macro'
```

Comment:
Have you found out why is this imported from `/macro` ?

Reply:
it's a babel bypass